src/Blocks/custom/carousel/carousel-style.scss

```diff
@@ -0,0 +1,80 @@
+@import '~swiper/swiper.scss';
+
+$block-carousel: (
+	pagination: (
+		fontSize: 18px,
+		fontWeight: bold,
+	),
+	button: (
+		size: 48px,
+		transition: 0.5s opacity ease-out 0.5s transform ease-out,
+		hoverScale: 1.1,
+		disabledOpacity: 0.2,
+		margin: 0 23px,
+		padding: 0,
+		border: 0,
+		background: none,
+	)
+);
+
+.block-carousel {
+	.swiper-wrapper {
+		cursor: grab;
+	}
+}
+
+.js-block-carousel {
```

Comment:
you should never put any styling on the js- selectors

https://infinum.com/handbook/books/frontend/bem/javascript